src/core/AddonsManager.cpp

```diff
@@ -141,6 +142,8 @@ void AddonsManager::loadUserScripts()
 			script->setEnabled(enabledScripts.value(scripts.at(i).fileName(), false));
 
 			m_userScripts[scripts.at(i).fileName()] = script;
+		} else {
+			Console::addMessage(QCoreApplication::translate("main", "Failed to find user script file: %1").arg(path), Console::OtherCategory, Console::WarningLevel);
```

Comment:
Please change "user script" to "User Script" for consistency with other such messages.